mesh/mesh.go

```diff
@@ -500,3 +500,66 @@ func (m *Mesh) GetATXs(atxIds []types.AtxId) (map[types.AtxId]*types.ActivationT
 	}
 	return atxs, mIds
 }
+
+// ActiveSetForLayerView - returns the active set size that matches the view of the contextually valid blocks in the provided layer
+func (m *Mesh) ActiveSetForLayerView(layer types.LayerID, layersPerEpoch uint16) (uint32, error) {
+
+	epoch := layer.GetEpoch(layersPerEpoch)
```

Comment:
why `layerPerEpoch`? Don't you mean `layer`? If yes, how come there is no test that failed?

Reply:
GetEpoch receives the number of layers in an epoch

Reply:
my bad